### PR TITLE
Map.1

### PR DESCRIPTION
**Pull request by olu144**
Add map to map.html

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -188,3 +188,9 @@ label {
   padding: 8px;
   font-family: Arial;
 }
+
+#map {
+  border: thin solid black;
+  height: 500px;
+  width: 500px;
+}
```
Comment (dborkan) [inline portfolio/src/main/webapp/style.css]: tiny nit, this file should end in a new line

**Review by dborkan — COMMENTED** [review]
Map code looks good, you'll just want to resolve any conflicts this PR has with the other branches

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/webapp/style.css]: addressed in "formatting"

**Comment by olu144** [comment]
Fixes from chart.5 have been merged

**Review by dborkan — APPROVED** [review]
👍 The new code here looks good to me.  You'll just want make sure when you merge that you resolve any conflicts with changes from other PRs (for example if you change the name of `drawAnimalChart` which was suggested in another PR)

**Pull request merged**